### ISSUE TITLE
publish new release versions to maven repo

### ISSUE THREAD

**Issue by aslom** [aslom/xpp3#3]
Need to figure out necessary steps, integration with CI like travis?

**Comment by psiroky** [comment]
Releasing into Maven Central should not be particularly hard after some initial setup is done. There is couple of links that may help : http://central.sonatype.org/pages/ossrh-guide.html http://www.sonatype.org/nexus/2015/01/08/deploy-to-maven-central-repository/.

**Comment by aslom** [comment]
@psiroky great I will follow the steps

**Comment by arnaud-penet** [comment]
Hi,

Any news on the release ?

Thx
Arnaud

**Comment by aslom** [comment]
I opened issue https://issues.sonatype.org/browse/OSSRH-31132

**Comment by arnaud-penet** [comment]
Hi, 

It seems the above issue is resolved, 
Do you think you can go ahead and release XPP3 ?

Thx
Arnaud

**Comment by aslom** [comment]
Yes - I got stuck on gpg keys - I am pretty sure I got one created long time ago but I can nto find it. If I no longer can find I will create new key ring.

**Comment by Jukka-A** [comment]
Hi,

I couldn't find new release. Can you do it?

**Comment by carlosame** [comment]
Hello,

I do not have my own artifacts at the Maven Central repository, so I'm not asking you to do that. However, I would appreciate if you did a release (a Github release or similar) of an XPP3 version with `javax.xml.namespace` removed.

I have been unlucky with building from Git (after cloning `https://github.com/aslom/xpp3.git` and running `mvn install` I get an `xpp3-1.1.7-SNAPSHOT.jar` of about 1.8KB). Also, the build instructions seem outdated (no `build.sh` script).

Thanks in advance.

**Comment by hick209** [comment]
Was this ever resolved? I'm curious whether this library is still active or not since I found an issue with the lib and wanted to understand how to setup my expectations on whether a fix would see the light of day if an issue is reported followed by a PR or if I should not waste my time on it (I'll report it regardless)

**Comment by carlosame** [comment]
> Was this ever resolved? I'm curious whether this library is still active or not since I found an issue with the lib and wanted to understand how to setup my expectations on whether a fix would see the light of day if an issue is reported followed by a PR or if I should not waste my time on it (I'll report it regardless)

None of the xpp3 projects/forks are active, but you could try with https://github.com/xmlpull-xpp3/xmlpull-xpp3. I deployed their 1.2 version on my Maven repository and works well with modular JDKs.
